apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictions/cell-population-predictions.component.html

```diff
@@ -0,0 +1,95 @@
+  tools predict different cell types and different percentages for the very same cell type. Hover over column headers
+  for details.
+</p>
+
+<hra-workflow-card title="Cell Population Predictions" class="table-container">
+  <hra-workflow-card-actions>
+    <button mat-icon-button aria-label="more icon button with a menu" aria-hidden="false" [matMenuTriggerFor]="options">
+      <mat-icon class="more-icon" fontIcon="more_vert"></mat-icon>
+    </button>
+
+    <!-- MENU -->
+    <mat-menu #options="matMenu">
+      @for (option of menuOptions; track option) {
+        @if (option.expandedOptions) {
+          <button
+            mat-menu-item
+            matRipple
+            matRippleColor="#201E3D14"
+            cdkOverlayOrigin
+            #tooltipTrigger="cdkOverlayOrigin"
+            (mouseover)="option.tooltipOpen = true"
+            (mouseout)="option.tooltipOpen = false"
+          >
+            <mat-icon>{{ option.icon }}</mat-icon>
+            {{ option.name }}
+            <mat-icon class="expand-arrow">arrow_right</mat-icon>
+          </button>
+
+          <ng-template
+            *ngIf="option.tooltip"
+            cdkConnectedOverlay
+            [cdkConnectedOverlayOrigin]="tooltipTrigger"
+            [cdkConnectedOverlayOpen]="option.tooltipOpen!"
+            [cdkConnectedOverlayPanelClass]="['tooltip-panel']"
+            [cdkConnectedOverlayPositions]="tooltipPosition"
+            [cdkConnectedOverlayViewportMargin]="8"
+            [cdkConnectedOverlayPush]="true"
+          >
+            <hra-tooltip-card [content]="option.tooltip"> </hra-tooltip-card>
+          </ng-template>
+        } @else {
+          <button mat-menu-item>
+            <mat-icon>{{ option.icon }}</mat-icon>
+            {{ option.name }}
+          </button>
+        }
+      }
+    </mat-menu>
+  </hra-workflow-card-actions>
+  <hra-workflow-card-extra>
+    <hra-delete-file-button (cancelLoad)="onDeleteButtonClick()" [fileName]="state.file.name"></hra-delete-file-button>
+  </hra-workflow-card-extra>
+
+  <!-- TABLE -->
+  <table mat-table [dataSource]="dataSource" matSort class="predictions-table">
+    <caption>
+      Table displaying cell population predictions.
+    </caption>
+    <ng-container matColumnDef="tool">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header sortActionDescription="Sort by Tool">Tool</th>
+      <td mat-cell *matCellDef="let element">{{ element.tool }}</td>
+    </ng-container>
+    <ng-container matColumnDef="modality">
+      <th mat-header-cell *matHeaderCellDef>Modality</th>
+      <td mat-cell *matCellDef="let element">{{ element.modality }}</td>
+    </ng-container>
+    <ng-container matColumnDef="percentage">
+      <th mat-header-cell *matHeaderCellDef>% Of Total</th>
+      <td mat-cell *matCellDef="let element">
+        {{ element.percentage < 0.0001 ? '< 0.01%' : (element.percentage | percent: '1.2-2') + '%' }}
```

Comment:
Doesn't the `percent` pipe add a `%` sign? Double check so that you don't get `1.23%%` etc.